posts/2020-09-01-jakarta-microprofile-beta-200010.adoc

```diff
@@ -0,0 +1,355 @@
+
+==== Timer
+The Time metric now tracks and reports the total elapsed time duration. See the Timer.java for more information in the Javadoc.
+
+==== REST metric
+A new REST.request.unmappedException.total metric that is backed by a counter metric has been introduced. Similar to the REST.request metric, there will be one unique metric for each REST endpoint identified by a class and method label. The new REST metric will count the amount of times the request ends in an unmapped exception. The REST.request metric corresponding to this REST endpoint will not record any values if an unmapped exception has occured.
+
+==== CDI Producer
+The @Metrics annotation will no longer support the method target (i.e it can not be annotated on a method). Additionally, it will not support usage with CDI Producers.
+
+==== Try it now 
+
+Enable Metrics 3.0 in the `server.xml`, along with any other features you're using.
+
+[source, xml]
+----
+<featureManager>
+    <feature>mpMetrics-3.0</feature>
+</featureManager>
+----
+
+More information:
+
+* link:https://github.com/eclipse/microprofile-metrics/releases/tag/3.0-RC1[MicroProfile Metrics 3.0 Release Page]
+
+* link:https://github.com/eclipse/microprofile-metrics/blob/3.0-RC1/spec/src/main/asciidoc/changelog.adoc[Release Notes]
+
+* link:https://download.eclipse.org/microprofile/microprofile-metrics-3.0-RC1/apidocs[Javadoc]
+
+[#health]
+=== MicroProfile Health 3.0
+
+MicroProfile Health 3.0 enables you to provide your own health check procedures to be invoked by Open Liberty, to verify the health of your microservice.
+
+MicroProfile Health allows services to report their health, and it publishes the overall health status to a defined endpoint. A service reports UP if it is available and reports DOWN if it is unavailable. MicroProfile Health reports an individual service status at the endpoint and indicates the overall status as UP if all the services are UP. A service orchestrator can then use the health statuses to make decisions.
+
+A service checks its own health by performing necessary self-checks and then reports its overall status by implementing the API provided by MicroProfile Health. A self-check can be a check on anything that the service needs, such as a dependency, a successful connection to an endpoint, a system property, a database connection, or the availability of required resources. MicroProfile offers checks for both liveness and readiness.
+
+In the mpHealth-3.0 feature for Open Liberty:
+
+The overall default Readiness status was changed to DOWN, with an empty response until all the deployed application(s) have been started. A new MicroProfile Config property (mp.health.default.readiness.empty.response=UP) is introduced to change the overall default Readiness check status to UP, during application start up, that do not have any user-defined health checks.
```

Comment:
I think `mp.health.default.readiness.empty.response=UP` should be in a code block

Reply:
Done (Sorry didn't notice this initially!)